s4e-web/src/app/utils/initializer/config.service.ts

```diff
@@ -1,48 +1,41 @@
+import { IRemoteConfiguration } from './../../app.configuration';
 import {Injectable} from '@angular/core';
-import {IConfiguration, IRemoteConfiguration} from '../../app.configuration';
-import {map, switchMap, tap} from 'rxjs/operators';
 import {HttpClient} from '@angular/common/http';
-import {SessionService} from '../../state/session/session.service';
+import environment from 'src/environments/environment';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
 
-@Injectable()
-export class S4eConfig implements IConfiguration {
-  backendDateFormat: string;
-  geoserverUrl: string;
-  geoserverWorkspace: string;
-  recaptchaSiteKey: string;
-  timezone: string;
+@Injectable({providedIn: 'root'})
+export class RemoteConfiguration {
+  isInitialized$: Observable<boolean>;
```

Comment:
I think it would be more consistent to have method eg.
```
isInitialized$() {
    return this._isInitialized$.asObservable();
}
```

Or at least make this public variable `readonly`;